application/forms/EventRuleConfigForm.php

```diff
@@ -244,14 +244,19 @@ public function createExternalSubmitButtons(): array
      */
     public function load(Rule $rule): void
     {
-        $this->populate([
+        $fields = [
             'id'                => $rule->id,
             'name'              => $rule->name,
-            'object_filter'     => $rule->object_filter,
-            'escalations'       => EventRuleConfigElements\Escalations::prepare(
+            'object_filter'     => $rule->object_filter
+        ];
+
+        if ($rule->rule_escalation->count() > 0) {
```

Comment:
You can note this down for the future, as it applies generally to all languages and frameworks:

> Don't count if you're not interested in the actual number of items.

In this case, you want to know whether there are any results or none. Our framework allows this by using this:

```php
$escalations = $query->execute();
if (! $escalations->hasResult()) {
```

Though, this will run the query explicitly. Don't forget to use the result in subsequent expressions, as otherwise the query is ran again and the advantage of using `hasResult` is gone.